models/User.js

```diff
@@ -2,9 +2,14 @@ const bcrypt = require("bcrypt");
 const mongoose = require("mongoose");
 
 const UserSchema = new mongoose.Schema({
-  userName: { type: String, unique: true },
+  userId: { type: String, unique: true },
```

Comment:
@intelagense Do we need to explicitly add a separate userID field? I think MongoDB/Mongoose will by default generate an *_id* field for every schema.

Reply:
This should be correct. @yuhwaa I can edit this out unless you disagree.

Reply:
That is right @JoeDravarol that we do not need to add one. I meant to use that auto-generated field for the ID of the user. It is not labeled clearly on the whiteboard.